src/controllers/controller.line.js

```diff
@@ -315,4 +330,50 @@ module.exports = DatasetController.extend({
 		model.borderWidth = valueOrDefault(options.hoverBorderWidth, options.borderWidth);
 		model.radius = valueOrDefault(options.hoverRadius, options.radius);
 	},
+
+	/**
+	 * Returns the stacks based on groups and line visibility.
+	 * @param {number} [last] - The dataset index
+	 * @returns {string[]} The list of stack IDs
+	 * @private
+	 */
+	_getStacks: function(last) {
```

Comment:
It looks like this is copied directly from the bar scale. I'm not sure it there's a preferred way of sharing the code, but I'd like to get other folks thoughts on this

Reply:
Hi @benmccann, 
I did it intentionally to maintain project "coerence". If this solution will be considered valid i can try to do a refactoring making those methods (_getStacks, getStackIndex) available in a common prototype (DatasetController?) and with the ability to choose axis (index or value) to work on.
let me know.
 regards
  Kolja